src/Sentry/Platforms/Native/Sentry.Native.targets

```diff
@@ -85,11 +85,25 @@
   <!-- Build the Sentry Native SDK (this only runs on local machines because in CI we expect the SDK to be
        built already on each native platform and fetched for the final .net build. -->
   <Target Name="_BuildSentryNativeSDK"
-    BeforeTargets="DispatchToInnerBuilds;BeforeBuild"
```

Comment:
The "when" now gets resolved by this target being "clamped" by the two new ones.